src/controllers/TeacherController.js

```diff
@@ -57,4 +57,17 @@ module.exports = {
       res.status(400).send(error);
     }
   },
+  async getBySubjectId(req, res) {
```

Comment:
добавить параметр next
```suggestion
  async getBySubjectId(req, res, next) {
```